### ISSUE TITLE
Bug: Can't add Galaxy Clusters to Events if Tag Editor permission is not granted

### ISSUE THREAD

**Issue by vincenzocaputo** [MISP/PyMISP#975]
It seems that adding a galaxy cluster to an event is not possible if these conditions are met:

- The Connector Tag associated with the galaxy does not exist
- The user does not have the Tag Editor permission

The PyMISP method `add_tag` returns the following message error:
```
Something went wrong (405): {'name': 'You do not have permission to use this functionality.', 'message': 'You do not have permission to use this functionality.', 'url': '/tags/add'}
{'errors': (405, {'name': 'You do not have permission to use this functionality.', 'message': 'You do not have permission to use this functionality.', 'url': '/tags/add'})}
```

However, the same user can attach the Galaxy Cluster to the Event via GUI, triggering the connector tag creation.

**Comment by SunsetR** [comment]
Hello, I encountered the same issue when updating events. Events can only be updated successfully when there are tag labels associated with the galaxy. Do you have any new solutions for this?🫡

**Comment by Rafiot** [comment]
This is odd it works via the GUI, not being to create tag when the user doesn't have the Tag Editor perms is working as it should be. 

Any idea @mokaddem @iglocska @righel ?

**Comment by vincenzocaputo** [comment]
> This is odd it works via the GUI, not being to create tag when the user doesn't have the Tag Editor perms is working as it should be.
> 
> Any idea @mokaddem @iglocska @righel ?

Yes, it works as it should but as far as I know at the moment the most convenient (and recommended) way to add a galaxy cluster to an event is to use connector tags (see also your response [here](https://github.com/MISP/PyMISPGalaxies/issues/2#issuecomment-421390448). 

Maybe the `add_galaxy` function should be used in these cases?

**Comment by iglocska** [comment]
> This is odd it works via the GUI, not being to create tag when the user doesn't have the Tag Editor perms is working as it should be.

Sounds like there (correctly) is an exception for that case in place, however, it also sounds like it's not adhered to everywhere. Definitely one to fix on MISP side.
